Extensions/ARAnalytics+Intercom.m

```diff
@@ -0,0 +1,50 @@
+//
+//  Created by Minh Tu Le 10/04/2014.
+//  Copyright (c) 2014 Wondermall inc. All rights reserved.
+//
+
+
+#import "ARAnalytics+Intercom.h"
+#import "Intercom.h"
+
+
+@implementation ARAnalytics (Intercom)
+
++ (void)presentMessageComposer {
+    [Intercom presentMessageComposer];
+}
+
++ (void)presentConversationList {
+    [Intercom presentConversationList];
+}
+
++ (void)setPreviewPosition:(ARPreviewPosition)previewPosition {
+    ICMPreviewPosition position = [self _translatePreviewPosition:previewPosition];
+    [Intercom setPreviewPosition:position];
+}
+
++ (void)setPreviewPaddingWithX:(CGFloat)x y:(CGFloat)y {
+    [Intercom setPreviewPaddingWithX:x y:y];
+}
+
++ (ICMPreviewPosition) _translatePreviewPosition:(ARPreviewPosition)previewPosition {
```

Comment:
The entire reason for this class is to abstract away the original library so, I don't want out app to use/import the original library directly.